### PR TITLE
feature: Add command to uninstall all servers

### PR DESCRIPTION
**Pull request by jcs090218**
Do `M-x lsp-uninstall-servers` to test it.

### CODE REVIEW

**Comment by jcs090218** [comment]
@yyoncho @ericdallo Any thoughts on this? :)

File: lsp-mode.el [hunk 1]
```diff
@@ -7839,11 +7839,30 @@ When prefix UPDATE? is t force installation even if the server is present."
...
+;;;###autoload
+(defun lsp-uninstall-servers ()
+  "Uninstall all installed servers."
+  (interactive)
+  (let* ((dir lsp-server-install-dir)
+         (servers (ignore-errors
+                    (directory-files dir t
+                                     directory-files-no-dot-files-regexp))))
+    (if (or (not (file-directory-p dir)) (zerop (length servers)))
+        (user-error "No server to uninstall")
```
Comment (ericdallo) [inline lsp-mode.el:7857]:
```suggestion
        (user-error "No servers to uninstall")
```

File: lsp-mode.el [hunk 2]
```diff
@@ -7839,11 +7839,30 @@ When prefix UPDATE? is t force installation even if the server is present."
...
+      (when (yes-or-no-p
+             (format "Servers to uninstall: %d (%s), proceed? "
+                     (length servers)
+                     (mapconcat (lambda (server)
+                                  (file-name-nondirectory (directory-file-name server)))
+                                servers " ")))
+        (mapc #'lsp-uninstall-server servers)
+        (message "Done uninstall all servers")))))
```
Comment (ericdallo) [inline lsp-mode.el:7865]:
```suggestion
        (message "All servers uninstalled")))))
```

**Review by ericdallo — APPROVED** [review]
Looks good!

**Pull request merged**